spec/cli_spec.rb

```diff
@@ -46,15 +47,12 @@
 
       context "syntax error movefile " do
         before do
-          allow(Wordmove::List).to receive(:start).and_raise(Psych::SyntaxError)
-        end
-
-        it { expect { subject }.to raise_error SystemExit }
-      end
-
-      context "load error movefile" do
-        before do
-          allow(Wordmove::List).to receive(:start).and_raise(LoadError)
+          # Ref. https://github.com/ruby/psych/blob/master/lib/psych/syntax_error.rb#L8
+          # Arguments for initialization: file, line, col, offset, problem, context
```

Comment:
Sorry that ``and_raise(Psych::SyntaxError)`` does not throw Psych::SyntaxError correctly.

Instead of Psych::SyntaxError, ``and_raise(Psych::SyntaxError)`` throws the ``ArgumentError``.
And the ``begin ... rescue`` statement handle exception as ``StandardError``.

So I changed to create Psych::SyntaxError instance via new method.

Reply:
It's awkward, but it happens and it's clear those magic numbers in `args` are there just to instantiate the exception object.

Thanks to have spotted it out, but all green from me ;)